website/docs/r/policy_edge_transport_node.html.markdown

```diff
@@ -0,0 +1,161 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_edge_transport_node"
+description: A resource to configure a Policy Edge Transport Node.
+---
+
+# nsxt_policy_edge_transport_node
+
+This resource provides a method for the management of a Policy Edge Transport Node.
+This resource is supported with NSX 9.0.0 onwards.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `site_path` - (Optional) The path of the site which the Edge Transport Node belongs to. `path` field of the existing `nsxt_policy_site` can be used here. Defaults to default site path.
+* `enforcement_point` - (Optional) The ID of enforcement point under given `site_path` to manage the Edge Transport Node. Defaults to default enforcement point.
+* `advanced_configuration` - (Optional) Array of additional specific properties for advanced or cloud-specific deployments in key-value format.
+  * `key` - (Required) Key.
+  * `value` - (Required) Value.
+* `appliance_config` - (Optional) Applicance configuration.
+  * `allow_ssh_root_login` (Optional) Allow root SSH logins.
+  * `dns_servers` - (Optional) DNS servers.
+  * `enable_ssh` - (Optional) Enable SSH.
+  * `enable_upt_mode` - (Optional) Enable Uniform Passthrough mode.
+  * `syslog_server` - (Optional) Syslog servers.
+    * `log_level` - (Optional) Log level to be redirected. Valid values are `DEBUG`, `INFO`, `NOTICE`, `WARNING`, `ERROR`, `CRITICAL`, `ALERT`, `EMERGENCY`. Default: `INFO`.
+    * `port` - (Optional) Syslog server port. Default: 514.
+    * `protocol` - (Optional) Syslog protocol. Valid values: `TCP`, `UDP`, `TLS`, `LI`, `LI_TLS`. Default: `UDP`.
+    * `server` - (Required) Server IP or fqdn.
+* `credentials` - (Optional) Username and password settings for the node.
+  * `audit_password` - (Optional) Node audit user password.
+  * `audit_username` - (Optional) CLI "audit" username.
+  * `cli_password` - (Required) Node cli password.
+  * `cli_username` - (Optional) CLI "admin" username.
+  * `root_password` - (Required) Node root user password.
+* `failure_domain_path` - (Optional) Path of the failure domain.
+* `form_factor` - (Optional) Appliance form factor. Valid values - 'SMALL', 'MEDIUM', 'LARGE', 'XLARGE'. The default value is 'MEDIUM'.
+* `hostname` - (Required) Host name or FQDN for edge node.
+* `management_interface` - (Optional) Applicable For LCM managed Node and contains the management interface info.
```

Comment:
judging from the code, this supports two assignments (I'm guessing v4 and v6) - lets mention this here